buildSrc/subprojects/release/src/main/kotlin/org/apache/jmeter/buildtools/jgit/dsl/Git.kt

```diff
@@ -0,0 +1,87 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package org.apache.jmeter.buildtools.jgit.dsl
+
+import org.apache.jmeter.buildtools.release.GitConfig
+import org.eclipse.jgit.api.Git
+import org.eclipse.jgit.api.TransportCommand
+import org.eclipse.jgit.transport.UsernamePasswordCredentialsProvider
+
+fun TransportCommand<*, *>.setCredentials(repo: GitConfig) =
```

Comment:
Is it possible to comment a bit what this class does ?

Reply:
Moved to https://github.com/vlsi/vlsi-release-plugins